### PR TITLE
Removed guava

### PR DESCRIPTION
**Pull request by satsen**
It wasn't necessary

### CODE REVIEW

File: src/com/pty4j/unix/NativePtyExecutor.java [hunk 1]
```diff
@@ -102,7 +103,7 @@ int exec_pty(String full_path, String[] argv, String[] envp, String dirpath, Str
    * @see <a href="https://man7.org/linux/man-pages/man2/ioctl_tty.2.html>ioctl_tty</a>
    */
   public static class WinSizeStructure extends Structure {
-    private static final List<String> FIELD_ORDER = ImmutableList.of("ws_row", "ws_col", "ws_xpixel", "ws_ypixel");
+    private static final List<String> FIELD_ORDER = Collections.unmodifiableList(Arrays.asList("ws_row", "ws_col", "ws_xpixel", "ws_ypixel"));
```
Comment (segrey) [inline src/com/pty4j/unix/NativePtyExecutor.java:106]: Please use `List.of`

File: src/com/pty4j/unix/UnixPtyProcess.java [hunk 2]
```diff
@@ -58,7 +57,7 @@ public UnixPtyProcess(@NotNull PtyProcessOptions options, boolean consoleMode) t
     myConsoleMode = consoleMode;
     myPty = new Pty(consoleMode, options.isUnixOpenTtyToPreserveOutputAfterTermination());
     myErrPty = options.isRedirectErrorStream() ? null : (consoleMode ? new Pty() : null);
-    String dir = MoreObjects.firstNonNull(options.getDirectory(), ".");
+    String dir = PtyUtil.firstNonNull(options.getDirectory(), ".");
```
Comment (segrey) [inline src/com/pty4j/unix/UnixPtyProcess.java:60]: Please use `Objects.requireNonNullElse`

**Review by segrey — CHANGES_REQUESTED** [review]

File: src/com/pty4j/unix/NativePtyExecutor.java (hunk 1 above)
Comment (segrey) [reply src/com/pty4j/unix/NativePtyExecutor.java:106]: Here and in other similar places in the PR.

File: src/com/pty4j/unix/UnixPtyProcess.java (hunk 2 above)
Comment (segrey) [reply src/com/pty4j/unix/UnixPtyProcess.java:60]: Here and in other similar places in the PR.

File: src/com/pty4j/unix/NativePtyExecutor.java (hunk 1 above)
Comment (satsen) [reply src/com/pty4j/unix/NativePtyExecutor.java:106]: @segrey I am aware of List.of, but it was added in Java 8. I thought you wanted to keep supporting old Java versions.

File: src/com/pty4j/unix/UnixPtyProcess.java (hunk 2 above)
Comment (satsen) [reply src/com/pty4j/unix/UnixPtyProcess.java:60]: @segrey Alright I was not aware of this method

File: src/com/pty4j/unix/NativePtyExecutor.java (hunk 1 above)
Comment (segrey) [reply src/com/pty4j/unix/NativePtyExecutor.java:106]: In fact, pty4j targets Java 11 (https://github.com/JetBrains/pty4j/blob/master/build.gradle#L58). This allows to implement some new java.lang.Process methods more efficiently.

Comment (satsen) [reply src/com/pty4j/unix/NativePtyExecutor.java:106]: @segrey That's cool, will change them then.

**Comment by satsen** [comment]
@segrey Alright I switched to JDK methods and squashed all commits, ready for merge

**Comment by segrey** [comment]
Thanks!

**Pull request merged**